src/main/java/stsjorbsmod/cards/cull/Ritual.java

```diff
@@ -0,0 +1,53 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.unique.RetainCardsAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.EquilibriumPower;
+import com.megacrit.cardcrawl.powers.IntangiblePlayerPower;
+import com.megacrit.cardcrawl.relics.RunicPyramid;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Ritual extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Ritual.class);
+
+    private static final CardRarity RARITY = CardRarity.COMMON;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 1;
+    private static final int INTANGIBLE = 1;
+    private static final int CARDS_TO_RETAIN = 1;
+    private static final int UPGRADE_CARDS_TO_RETAIN = 1;
+
+    public Ritual() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        magicNumber = baseMagicNumber = CARDS_TO_RETAIN;
+        urMagicNumber = baseUrMagicNumber = INTANGIBLE;
+        exhaust = true;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        addToBot(new ApplyPowerAction(p, p, new IntangiblePlayerPower(p, urMagicNumber)));
+        // This makes sure that choice window does not pop-up when this is the last card played from hand
+        if (!(p.hand.size() == 1 && p.hand.group.get(0) == this) && !p.hand.isEmpty() && !AbstractDungeon.player.hasRelic(RunicPyramid.ID) && !AbstractDungeon.player.hasPower(EquilibriumPower.POWER_ID)) {
```

Comment:
I think maybe it would be safest to move this whole if block into a separate action; that way, if there's anything that triggers a card draw somehow in response to the intangible being applied (or in response to phantasm damaging/killing something after intangible is applied), we'll properly respect the updated hand size.